usaspending_api/awards/management/commands/load_subawards.py

```diff
@@ -0,0 +1,194 @@
+import logging
+import os
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+from functools import partial
+from psycopg2.sql import Identifier, Literal, SQL
+from usaspending_api.common.helpers.sql_helpers import convert_composable_query_to_string, get_connection
+from usaspending_api.common.helpers.timing_helper import Timer
+
+
+logger = logging.getLogger("console")
+
+
+# All of our timers log to logger.info and logger.error.
+Timer = partial(Timer, success_logger=logger.info, failure_logger=logger.error)
+
+
+class Command(BaseCommand):
+    help = "Load subawards from Broker into USAspending."
+
+    def __init__(self, *args, **kwargs):
+        self.full_reload = False
+        self.log_sql = False
+        super().__init__(*args, **kwargs)
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--full-reload",
+            action="store_true",
+            help="Empties the USAspending subaward and broker_subaward tables before loading."
+        )
+
+        parser.add_argument(
+            "--sql",
+            action="store_true",
+            help="For debugging.  Log all SQL statements.  More verbose than you might expect."
+        )
+
+    def handle(self, *args, **options):
+
+        # Pick out just the last bit of the module name (which should match the
+        # file name minus the extension).
+        module_name = __name__.split(".")[-1]
+        with Timer(module_name):
+
+            self.full_reload = options["full_reload"]
```

Comment:
not options["full-reload"]?

Reply:
Not even a little bit... https://docs.python.org/dev/library/argparse.html#dest

> Any internal - characters will be converted to _ characters to make sure the string is a valid attribute name.